inst/rmarkdown/templates/shower/template.yaml

```diff
@@ -1,4 +1,4 @@
-name: Shower Presentation (HTML)
+name: Shower Presentation
 description: >
-  Template for HTML presentations based on Shower (http://shwr.me/)
+   Template for HTML presentations based on Shower (http://shower.me/)
```

Comment:
argh. yeah. i that first as the markdown viewer was whining abt it. fixing


On Thu, Mar 24, 2016 at 4:29 PM, Gábor Csárdi <notifications@github.com>
wrote:

> In inst/rmarkdown/templates/shower/template.yaml
> <https://github.com/MangoTheCat/rmdshower/pull/17#discussion_r57382611>:
>
> >  description: >
> > -  Template for HTML presentations based on Shower (http://shwr.me/)
> > +   Template for HTML presentations based on Shower (http://shower.me/)
>
> I don't think this URL is correct.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MangoTheCat/rmdshower/pull/17/files/bc51052bfc18b3ab5221668642e855e57a35186d#r57382611>
>